test/functional/test-3p-frame.js

```diff
@@ -124,32 +124,65 @@ describe('3p-frame', () => {
     const width = window.innerWidth;
     const height = window.innerHeight;
     const amp3pSentinel = iframe.getAttribute('data-amp-3p-sentinel');
-    const fragment =
-        '{"testAttr":"value","ping":"pong","width":50,"height":100,' +
-        '"type":"_ping_"' +
-        ',"_context":{"referrer":"http://acme.org/",' +
-        '"canonicalUrl":"https://foo.bar/baz",' +
-        '"pageViewId":"' + docInfo.pageViewId + '","clientId":"cidValue",' +
-        '"location":{"href":"' + locationHref + '"},"tagName":"MY-ELEMENT",' +
-        '"mode":{"localDev":true,"development":false,"minified":false,' +
-        '"test":false,"version":"$internalRuntimeVersion$"}' +
-        ',"hidden":false' +
-        ',"startTime":1234567888' +
-        ',"amp3pSentinel":"' + amp3pSentinel + '"' +
-        ',"initialIntersection":{"time":1234567888,' +
-        '"rootBounds":{"left":0,"top":0,"width":' + width + ',"height":' +
-        height + ',"bottom":' + height + ',"right":' + width +
-        ',"x":0,"y":0},"boundingClientRect":' +
-        '{"width":100,"height":200},"intersectionRect":{' +
-        '"left":0,"top":0,"width":0,"height":0,"bottom":0,' +
-        '"right":0,"x":0,"y":0}}}}';
```

Comment:
After I wrote my code, I ran the tests and noticed this one failing. So I tried adding ing `AMP_CONFIG: {}` in that  string, and was missing a comma. Took forever to run the tests and find out I was missing a comma. So, I re-wrote this string to be an object that gets stringified when necessary. That way it's easier to read and bugs are easier to catch